backend/LexCore/Entities/Organization.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace LexCore.Entities;
+
+public class Organization : EntityBase
+{
+    public required string Name { get; set; }
+    public required List<OrgMember> Members { get; set; }
```

Comment:
Will we want to allow sub-organizations, e.g. MSEAG is a sub-organization of SIL? Or is the thinking that these orgs will be small-scale, e.g. the Foo language team is one org and the Bar language team is a different org, and then anyone working on the Foo language team can see all projects created by other Foo team members but not Bar?

Reply:
addressed in comment in pr. Out of scope for now.